packages/daimo-expo-enclave/android/build.gradle

```diff
@@ -37,7 +37,7 @@ buildscript {
 
 // Creating sources with comments
 task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
```

Comment:
whats the difference between classifier and archiveClassifier / how does it change the output jarfile?